server-ui/src/KeyView.tsx

```diff
@@ -141,23 +140,26 @@ export const KeyView = (props: {
     return defaultStyle;
   }
 
+  if (kindEditor.getRendereditordropdownwithfield() === "") {
+    console.log("using default rendering for that type");
```

Comment:
Remove this log message.

Reply:
Didn't remove log message so it is consistent with surrounding code, but did remove the accidental duplicate I added of this check